config/routes.rb

```diff
@@ -36,9 +36,9 @@
 
   get  '/convert',    to: 'converts#new'
 
-  get '/search',        to: redirect('/games') # all deprecated, use GET /games
-  get '/search/:q',     to: redirect('/games?q=%{q}')
-  get '/search(?q=:q)', to: redirect('/games?q=%{q}')
+  get '/search',        to: 'search#index'
+  get '/search/:q',     to: 'search#index'
+  get '/search(?q=:q)', to: 'search#index'
```

Comment:
It would be nice if one of these
```ruby
get '/search/:q'
get '/search(?q=:q)'
```
redirected to the other, so we can have one de facto URL style for the search results page. I have no strong preference for which one is the standard.

Reply:
I personally like `get '/search(?q=:q)'` if that's cool with you.